src/i18n/pl.js

```diff
@@ -0,0 +1,102 @@
+export default {
+  error: {
+    forgotPassword: {
+      "too_many_requests": "Przekroczyłeś limit prób zmiany hasła. Odczekaj chwilę, zanim spróbujesz ponownie.",
+      "lock.fallback": "Przykro nam, coś poszło nie tak podczas próby zmiany hasła."
+    },
+    login: {
+      "blocked_user": "Ten użytkownik jest zablokowany.",
+      "invalid_user_password": "Niepoprawne dane logowania.",
+      "lock.fallback": "Przykro nam, coś poszło nie tak podczas próby logowania.",
+      "lock.invalid_code": "Niepoprawny kod.",
+      "lock.invalid_email_password": "Niepoprawny email lub hasło.",
+      "lock.invalid_username_password": "Niepoprawna nazwa użytkownika lub hasło.",
+      "lock.network": "Kontakt z serwerem nie powiódł się. Sprawdź swoje połączenie i spróbuj ponownie.",
+      "lock.popup_closed": "Okno zamknięte. Spróbuj ponownie.",
+      "lock.unauthorized": "Autoryzacja nie została udzielona. Spróbuj ponownie.",
+      "password_change_required": "Musisz zmienić swoje hasło, poniewasz logujesz się po raz pierwszy lub Twoje hasło wygasło.", // TODO: verify error code
+      "password_leaked": "Ta nazwa użytkownika została zablokowana, ponieważ Twoje hasło wyciekło w innym serwisie. Wysłaliśmy do Ciebie email z instrukcją, w jaki sposób ją odblokować.",
+      "too_many_attempts": "Twoje hasło zostało zablokowane po zbyt wielu nieudanych próbach logowania."
+    },
+    passwordless: {
+      "bad.email": "Adres email nie jest poprawny",
+      "bad.phone_number": "Numer telefonu nie jest poprawny",
+      "lock.fallback": "Przykro nam, coś poszło nie tak"
+    },
+    signUp: {
+      "invalid_password": "Hasło nie jest poprawne.",
+      "lock.fallback": "Przykro nam, coś poszło nie tak podczas próby rejestracji.",
+      "password_dictionary_error": "Hasło jest zbyt oczywiste.",
+      "password_no_user_info_error": "Hasło oparte jest na informacjach o użytkowniku.",
+      "password_strength_error": "Hasło jest zbyt słabe.",
+      "user_exists": "Użytkownik już istnieje.",
+      "username_exists": "Nazwa użytkownika już istnieje."
+    }
+  },
+  success: { // success messages show above the form or in a confirmation pane
+    logIn: "Logowanie poprawne, dziękujemy.",
+    forgotPassword: "Wysłaliśmy do Ciebie email z informacją, jak zresetować hasło.",
```

Comment:
Changed to "Wysłaliśmy do Ciebie email z instrukcją, jak zresetować hasło."

Polish note: fraza "z instrukcją resetowania" brzmi niezbyt stylistycznie.